extra-utils/mcfly/autobuild/postinst

```diff
@@ -0,0 +1,13 @@
+echo -e "
+mcfly will be loaded automatically the next time you start the bash shell, or if you are using the bash shell and loading it immediately, type:
+
+. /etc/profile.d/mcfly.bash
+
+If you are using Zsh and want to load it automatically on next startup, add the following statement to your `~/.zshrc` :
```

Comment:
If you are using Zsh and want to load McFly automatically, please append the following line to `~/.zshrc` :